### PR TITLE
Add napari-graph recipe

### PR DESCRIPTION
**Pull request by goanpeca**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- ~[ ] If static libraries are linked in, the license of the static library is packaged.~
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by goanpeca** [comment]
We are in the process of creating conda packages for [napari plugins](https://www.napari-hub.org/) and related dependencies. Once this recipe passes all the checks, every time a new release goes into PyPI, a PR will be created automatically, with usually very minor need for changes so the process of maintaining this moving forward is small. Also the `@conda-forge/napari` team will be available to help as needed.

You can read a bit more about conda-forge at https://conda-forge.org/#contribute

---

@jaimergp, @JoOkuma do you agree with being added as maintainers?

**Comment by jni** [comment]
🙋 yes please! 😂 Actually I think we should also add @dragadoncila @kevinyamauchi and @kephale.

**Comment by kevinyamauchi** [comment]
I agree to be a maintainer.

**Comment by kephale** [comment]
I agree too!

**Comment by goanpeca** [comment]
This one is ready for review @conda-forge/staged-recipes :)

File: recipes/napari-graph/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{% set name = "napari-graph" %}
+{% set version = "0.1.0" %}
```
Comment (jaimergp) [inline recipes/napari-graph/meta.yaml:2]:
```suggestion
{% set version = "0.2.0" %}
```

File: recipes/napari-graph/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-graph-{{ version }}.tar.gz
+  sha256: 485ef8d2695593e5f418f144d10420859f4075942a4c4bd4f07f5222ea17eb82
```
Comment (jaimergp) [inline recipes/napari-graph/meta.yaml:10]:
```suggestion
  sha256: fb93c22659d8910168197e2c69693dffb49cc84e09b937fd3764a2aac7c757fe
```

File: recipes/napari-graph/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+    - setuptools-scm
+  run:
+    - numba
```
Comment (jaimergp) [inline recipes/napari-graph/meta.yaml:23]:
```suggestion
    - networkx
```

File: recipes/napari-graph/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    - napari_graph
+  commands:
+    - pip list
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/napari/napari-graph
+  summary: Fast editable graphs in Python and numba
```
Comment (jaimergp) [inline recipes/napari-graph/meta.yaml:39]:
```suggestion
  summary: Efficient graph data structures and algorithms for fast slicing, visualization, and editing
```

**Pull request merged**